content/foundation/voting.md

```diff
@@ -109,7 +109,8 @@ expressing is Boolean: 'I approve/do not approve of this change.'
 ### Votes on package releases  {#ReleaseVotes}
 
 Votes on whether a package is ready to release use 
-[majority approval](glossary.html#MajorityApproval) -- 
+[majority approval](glossary.html#MajorityApproval),
+plus a minimum quorum of three positive votes,
 i.e. at least three PMC members must vote affirmatively
 for release, and there must be more positive than negative votes.
```

Comment:
This now states three positive votes in two different ways.
```
plus a minimum quorum of three positive votes,
i.e. at least three PMC members must vote affirmatively
```

Reply:
Since they're the same thing, it is what we expected?

Or we can cut the long sentence as:

> Votes on whether a package is ready to release use 
> [majority approval](glossary.html#MajorityApproval),
> plus a minimum quorum of three positive votes.
> That is, at least three PMC members must vote affirmatively
> for release, and there must be more positive than negative votes.
> **Releases may not be vetoed.** 

Or just states it in the same way:

> Votes on whether a package is ready to release use 
> [majority approval](glossary.html#MajorityApproval),
> plus at least three binding +1 votes,
> i.e., at least three PMC members must vote affirmatively
> for release, and there must be more positive than negative votes.
> **Releases may not be vetoed.**

Reply:
Updated.